ui/apps/moderation/src/components/content-card/content.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import moment from 'moment';
```

Comment:
Do we really need moment? I think we have day.js

Reply:
A couple of components outside design system use moment. I can clean that up to use dayjs and remove moment entirely, in a separate PR